packages/plugins/i18n/server/controllers/content-types.js

```diff
@@ -32,7 +38,7 @@ module.exports = {
 
     const entity = await strapi
       .query(model)
-      .findOne({ where: params, populate: ['localizations'] });
+      .findOne({ where: params, populate: [...nonLocalizedMediaAttributes, 'localizations'] });
```

Comment:
Now that I look back at the code I wonder if the populate can break if there are non relational attributes here, I'm going to check that

Reply:
It breaks the code, will change that